### ISSUE TITLE
repeated error message / thermostat control not working

### ISSUE THREAD

**Issue by dpdpdpdpdp** [chrisjshull/homebridge-nest#365]
For the last 2-3 days, my Nest thermostat shows up in the Home app and I can manipulate the temperature control there, but it doesn't actually do anything.  The following error message repeats every second or two:

[11/02/2021, 07:57:56] [Nest] Nest API call to subscribe to device settings updates returned an error: ENOTFOUND

When I use my browser or the iOS Nest app, everything works fine.

**Comment by samemory** [comment]
I also have had this issue. From the Apple home app, it would appear everything is working fine but in reality, it doesn't actually update the Nest. I first discovered this when adjusting the temperature but when I walked by my Nest I noticed it hadn't changed and had a current temperature that was different from the Apple Home app.
Restarting the homebridge server has fixed this each time and it seems to happen about every two weeks now. Makes it so I cannot rely on homekit to know the actual temp in my house or make any changes.
